clients/web/src/views/widgets/EditFolderWidget.js

```diff
@@ -28,8 +28,13 @@ girder.views.EditFolderWidget = girder.View.extend({
             text: this.folder ? this.folder.get('description') : '',
             prefix: 'folder-description',
             placeholder: 'Enter a description',
+            parent: this.folder,
+            allowedExtensions: ['png', 'jpg', 'jpeg'],
```

Comment:
We could add anything that can be rendered in an `img` tag, I just picked this set for now. Do you think we should add other extensions?